ui/k-info-panel/switch_info.kytos

```diff
@@ -61,7 +63,43 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+     bt_enable(){
+       var notification = {
+         title: "",
+         description: "",
+         icon: "gear",
+       }
+       $.ajax({
+         type:"POST",
+         url: this.$kytos_server_api + "kytos/topology/v3/switches/" + this.metadata.dpid
+              + "/enable",
+         async: true,
+         success: function(message){
```

Comment:
@Alopalao, could you handle the case where a request might fail? Also, even though the `success` callback works great here, I'd like to ask you to use the `done` callback instead, we've got both usages in the code base, but the team is leaning towards to use [`done` and `fail`](https://github.com/kytos-ng/mef_eline/blob/master/ui/k-toolbar/main.kytos#L174-L175) callbacks to handle both cases, check out the link for an example of how it was implemented in a UI component on `mef_eline`